frontend/public/index.html

```diff
@@ -0,0 +1,69 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="theme-color" content="#000000">
+  <meta http-equiv="Content-Security-Policy" content="default-src 'self'
```

Comment:
embedding thr CSP into index.html is not ideal, but until S3/CloudFront support this natively it is better than nothing.